docs/en/Background-Jobs-Hangfire.md

```diff
@@ -39,5 +39,50 @@ public class YourModule : AbpModule
 ````
 
 ## Configuration
+You can install any storage for Hangfire. The most common one is SQL Server (see the [Hangfire.SqlServer](https://www.nuget.org/packages/Hangfire.SqlServer) NuGet package).
 
-TODO...
\ No newline at end of file
+After you have installed these NuGet packages, you need to configure your project to use Hangfire.
+
+1.First, we change the `<YourProjectName>HttpApiHostModule` class to add Hangfire configuration of the storage and connection string in the `ConfigureServices` method:
+
+
+````csharp
+  public override void ConfigureServices(ServiceConfigurationContext context)
+  {
+      var configuration = context.Services.GetConfiguration();
+      var hostingEnvironment = context.Services.GetHostingEnvironment();
+
+      //... other configarations 
+
+      ConfigureHangfire(context, configuration);
+  }
+
+  private void ConfigureHangfire(ServiceConfigurationContext context, IConfiguration configuration)
+  {
+      context.Services.AddHangfire(config =>
+      {
+          config.UsePostgreSqlStorage(configuration.GetConnectionString("Default"));
+      });
+  }
+````
+
+2. We need to add `UseHangfireServer` call in the Configure method in `Startup` class
```

Comment:
We can call `UseXXX()` in the module's `OnApplicationInitialization `method.

https://github.com/abpframework/abp/blob/dev/templates/app/aspnet-core/src/MyCompanyName.MyProjectName.Web/MyProjectNameWebModule.cs#L189

Reply:
Added changes with this commit : https://github.com/suravi999/abp/commit/0e70f9e9b6fc8fc19a8e88a4eaba8eccf9180e40